scico/linop/_linop.py

```diff
@@ -97,33 +97,35 @@ def valid_adjoint(
     eps: Optional[float] = 1e-7,
     key: Optional[PRNGKey] = None,
 ) -> Union[bool, float]:
-    r"""Check whether :class:`.LinearOperator` `AT` is the adjoint of `A`.
+    r"""Check whether :class:`.LinearOperator` :math:`\mathsf{AT}` is the adjoint of :math:`\mathsf{A}`.
 
     The test exploits the identity
 
     .. math::
       \mathbf{y}^T (A \mathbf{x}) = (\mathbf{y}^T A) \mathbf{x} =
       (A^T \mathbf{y})^T \mathbf{x}
 
-    by computing :math:`\mathbf{u} = A \mathbf{x}` and
-    :math:`\mathbf{v} = A^T \mathbf{y}` for random :math:`\mathbf{x}`
-    and :math:`\mathbf{y}` and confirming that :math:`\| \mathbf{y}^T
-    \mathbf{u} - \mathbf{v}^T \mathbf{x} \|_2 < \epsilon` since
+    by computing :math:`\mathbf{u} = \mathsf{A}(\mathbf{x})` and
+    :math:`\mathbf{v} = \mathsf{AT}(\mathbf{y})` for random :math:`\mathbf{x}`
+    and :math:`\mathbf{y}` and confirming that
 
     .. math::
-      \mathbf{y}^T \mathbf{u} = \mathbf{y}^T (A \mathbf{x}) =
-      (A^T \mathbf{y})^T \mathbf{x} = \mathbf{v}^T \mathbf{x}
+      \frac{\| \mathbf{y}^T \mathbf{u} - \mathbf{v}^T \mathbf{x} \|_2}
+      {\max \left\{ \| \mathbf{y}^T \mathbf{u} \|_2,
```

Comment:
I suggest using absolute value rather than norm as these are scalar quantities.

Reply:
I think I fixed it. Can someone double check? I seem to be having issues with building the docs.

Reply:
The CI builds the docs automatically, look below this comment, "show all checks" -> docs/readthedocs.org:scico, click "details"

Reply:
They still look like norms, need to change `\|` to `|` or `\lvert` `\rvert`.

Reply:
> The CI builds the docs automatically, look below this comment, "show all checks" -> docs/readthedocs.org:scico, click "details"

Thanks. Good to know. Made the change using `|`